### DISCUSSION TITLE
Feature request or modification

### DISCUSSION THREAD

**Discussion by Lynnolium** [dnSpyEx/dnSpy#108] [Ideas]
I use DNSpy a lot to copy IL code from one method to another in different 'modules' . The current process is to open both modules and (methods) in the same session of DNSpy. Then go to Edit Method Body select and copy all the IL code from the Instructions tab. This copied code is then pasted into the destination method for further editing. The issue I would like to have fixed/adjusted is for the data (IL code) on the clipboard to be retained and transferrable between different active sessions of DNSpy. IE open two sessions and copy code from one session to another. This process is available using the Edit Class(C#) function.

On a related issue and maybe a possible solution to the above. Being able to just add/copy the whole method from one Class or Module to another and then saving/compiling the module would work too even if its only in the same session.

**Comment by dem0nk1d** [comment]
yes, yes, and yes!
